public/javascripts/conversation.js

```diff
@@ -318,4 +332,26 @@ $(function() {
         $('#conversation').collapse('show');
         event.preventDefault();
     });
+
+    $('#input-form').on('keydown', function(event) { // up button is pressed (For issue #172)
+      if (event.keyCode == 38) {  //Up
```

Comment:
Please use `===` exclusively.
(I'm surprised eslint did not complain about this).